Packs/Thycotic/Integrations/Thycotic/Thycotic.py

```diff
@@ -465,8 +454,26 @@ def secret_rpc_changepassword_command(client, secret_id: str = '', newPassword:
     )
 
 
+def test_module(client) -> str:
+    if client._is_fetch_credential:
+        if len(client._credential_objects) == 0:
+            return "Enter secret name."
+
+        if client.getCredentials() == "":
+            return "Failed search secret name for sync"
+
+    if client._token == '':
+        return "Failed to get authorization token. Check you credential and access to Secret Server.'"
+
+    return "ok"
+
+
 def fetch_credentials(client):
-    credentials = client.getCredentials()
+    credentials = []
+    jsonCredential = client.getCredentials()
+    if jsonCredential != "":
+        credentials.append(json.loads(jsonCredential))
```

Comment:
1. For some reason it seems you are calling `json.loads()`on a dict, in the  - is that intentional?\
2. It seems like the `credentials` array currently will always contain only one credential - why is that?

Reply:
It still seems like you aren't using that `args`.
Is this ready for review or are you still working on it? I wasn't sure